canonical-web-and-design/jaas-dashboard
# Convert Button to Typescript

## Done

Convert Button to Typescript

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Check tests pass and all is as expected
